src/components/Modals/DetailModal/DetailModal.tsx

```diff
@@ -446,7 +446,18 @@ class DetailModalInternal extends Component<IDetailModalInternal> {
         this.props.orderOfCases.forEach((caseItem) => {
             if (caseItem in paradigmArray.cases) {
                 const caseName = t(`case${caseItem[0].toUpperCase()}${caseItem.slice(1)}`);
-                const tableRow = [`${caseName}@b`];
+                let caseTips = '';
+                switch (caseItem) {
+                    case 'nom': caseTips = 'kto? čto?'; break;
+                    case 'acc': caseTips = 'kogo? čto?'; break;
+                    case 'gen': caseTips = 'kogo? čego?'; break;
+                    case 'dat': caseTips = 'komu? čemu?'; break;
+                    case 'ins': caseTips = 'kym? čim?'; break;
+                    case 'loc': caseTips = 'kom? čem?'; break;
+                    case 'voc': caseTips = 'hej!'; break;
+                    default: caseTips = `test_${caseItem}`;
```

Comment:
Dlja čego tutoj placeholder, `test_${caseItem}` jest potrěbny?

Reply:
Ja igraju se... Možeš poka ne gledeti, to jest draft. Mně bude potrěbna tvoja pomoč sdělati lokalny nodejs server, aby testovati svoje igrački ne tut.